dotcom-rendering/src/web/layouts/FullPageInteractiveLayout.tsx

```diff
@@ -328,23 +328,25 @@ export const FullPageInteractiveLayout = ({
 				backgroundColour={palette.background.article}
 				element="main"
 			>
-				<article>
-					<Renderer
-						format={format}
-						elements={
-							CAPIArticle.blocks[0]
-								? CAPIArticle.blocks[0].elements
-								: []
-						}
-						host={host}
-						pageId={CAPIArticle.pageId}
-						webTitle={CAPIArticle.webTitle}
-						ajaxUrl={CAPIArticle.config.ajaxUrl}
-						switches={CAPIArticle.config.switches}
-						isAdFreeUser={CAPIArticle.isAdFreeUser}
-						isSensitive={CAPIArticle.config.isSensitive}
-					/>
-				</article>
+				<div id="maincontent" tabIndex={-1}>
+					<article>
```

Comment:
Does this change break any contracts we might have with interactive content? These articles are each written slightly differently and sometimes there is javascript which reads the dom expecting a certain structure.

Unfortunately, we have no visibility on what these contracts are but it's possible adding in this div could create issues. To mitigate this, would it work if we move the id and tabIndex onto the `article` element instead?